mypyc/irbuild/ll_builder.py

```diff
@@ -649,20 +653,23 @@ def add_bool_branch(self, value: Value, true: BasicBlock, false: BasicBlock) ->
         self.add(Branch(value, true, false, Branch.BOOL_EXPR))
 
     def call_c(self,
-               function_name: str,
+               desc: CFunctionDescription,
                args: List[Value],
-               line: int,
-               result_type: Optional[RType]) -> Value:
```

Comment:
I think that this is still useful to have. This would be the type the result should be coerced into, which can be different from the return type of the called function.